RadarSDK/Radar.m

```diff
@@ -564,11 +564,12 @@ + (void)startTripWithOptions:(RadarTripOptions *)tripOptions
                                              if (status == RadarStatusSuccess) {
                                                  [RadarSettings setTripOptions:tripOptions];
 
-                                                 if (Radar.isTracking) {
+                                                 if (Radar.isTracking && ![RadarSettings rampedUp]) {
```

Comment:
Is this so that ramped up tracking overrides trackingOptions that are used when a trip starts?  If so we may want to take the approach of having people set the ramp up options on both on trip and off trip tracking options to keep this simple.  Then I think we could remove this change?

Reply:
Ahhh, with the changes to collapse everything into the existing tracking options (no new `rampedUpTrackingOption` preset), we can revert this change, nice catch.